pages/common/shellcheck.md

```diff
@@ -1,33 +1,36 @@
 # shellcheck
 
-> Shell script static analysis tool.
-> Check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
+> Statically check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
```

Comment:
Let me take this opportunity to adequate the description to our style, what do you guys think of this?